File: in_Python/0961 N-Repeated Element in Size 2N Array.py
class Solution(object):
    def repeatedNTimes(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        t = []
        res = 0
        for i in A:
            if i not in t: 
                t.append(i)
            else:
                res = i
                break
        return res
        

File: in_Java/0961 N-Repeated Element in Size 2N Array.java
class Solution {
    public int repeatedNTimes(int[] A) {
        Set<Integer> h = new HashSet<>();
        int res = A[0];
        for(int i : A){
            if(!h.contains(i)) h.add(i);
            else{
                res = i;
                break;
            }
        }
        return res;
    }
}